assets/style.css

```diff
@@ -255,24 +223,11 @@ header {
   cursor: pointer;
 }
 
-/* probably delete from final build */
-/* .bulma-fa-mixin {
-  background-color: transparent;
-} */
-
-
 /* results tiles section */
-
 .columns {
   justify-content: center;
 }
 
-
-/* delete from final build */
-/* .colorTile p {
-  font-size: 1.25rem;
-} */
-
 .column1,
```

Comment:
These could be given a shared class to set the same width for all.

Reply:
I can do that if you're fine with me changing the HTML too